src/application-navigation/index.js

```diff
@@ -0,0 +1,10 @@
+import {
+  titleConfigPropType, navigationItemsPropType, extensionItemsPropType, utilityItemsPropType, userConfigPropType,
+} from 'terra-application-navigation/lib/utils/propTypes';
+import ApplicationNavigation from './ApplicationNavigation';
+
+export default ApplicationNavigation;
+export {
+  titleConfigPropType, navigationItemsPropType, extensionItemsPropType, utilityItemsPropType, userConfigPropType,
```

Comment:
why export the `terra-application-navigation` prop types instead of the proptypes for `./ApplicationNavigation`?

Reply:
The different propType shapes aren't exported from ./ApplicationNavigation. We import them there, we use them there, but we're not exporting them from there because this index file is a more likely consumption point.